src/plugins/git/gitUtils.js

```diff
@@ -4,6 +4,7 @@ import {execFileSync} from "child_process";
 import fs from "fs";
 import mkdirp from "mkdirp";
 import path from "path";
+import process from "process";
```

Comment:
`process` is on the global object; no need to import.

Reply:
Done, thanks.